extra-utils/daemonize/spec

```diff
@@ -0,0 +1,4 @@
+VER=1.7.8
+GITSRC="https://github.com/bmc/daemonize.git"
```

Comment:
Please use the new ACBS [spec format](https://wiki.aosc.io/developer/packaging/acbs/spec-format/), here's a rewrite of this file, for your reference.

```
VER=1.7.8
SRCS="git::commit=tags/release-$VER::https://github.com/bmc/daemonize.git"
CHKSUMS="SKIP"
```

Reply:
ack

Reply:
fixed